front/src/modules/scenario/components/ScenarioExplorer/ScenarioExplorer.tsx

```diff
@@ -176,7 +164,7 @@ const ScenarioExplorer = ({
                 </span>
 
                 <span className="scenario-explorator-card-head-scenario-traincount">
-                  {timetable && trainCount(timetable.train_ids)}
+                  {timetable && trainCount(timetable.results.map((r) => r.id))}
```

Comment:
TBH I think we can just drop the `trainCount()` function. `timetable.results.length` should work fine.